tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -593,3 +594,14 @@ RHODS Notification Drawer Should Not Contain
     Click Element    xpath=//*[contains(@class,'notification-badge')]
     Page Should Not Contain  text=${message}
     Click Element    xpath=//*[contains(@class,'notification-badge')]
+
+Check Doc And Quick Start Link In Enabled Page Tile
+    [Documentation]    Checks for the Doc and Quick Start Link in the tile of enabled page
+    ...                ${id} is the id of the tile
+    [Arguments]    ${id}
+    Click Button    //article[@id="${id}"]/div/div/div/button
+    ${href} =    Get Element Attribute    //a[contains(text(),"View documentation")]    href
+    Check HTTP Status Code    ${href}
```

Comment:
could you also click on the Doc link? to check if the link is clickable

Reply:
You can click to check if the link works and then close the window to go back